qcodes/instrument/parameter.py

```diff
@@ -1364,6 +1364,13 @@ def get(self):
             else:
                 return state['value']
 
+    def get_timestamp(self):
```

Comment:
is this a correct place for such a method? @jenshnielsen also with the fact that `_latest` lives inside parameter.

Reply:
I think the GetLatest thing is a somewhat strange construction but given that it exists I think this is ok